src/OpenApi/gen/XmlComments/XmlComment.cs

```diff
@@ -197,6 +202,35 @@ private static void ResolveCrefLink(Compilation compilation, XNode node, string
         }
     }
 
+    /// <summary>
+    /// Resolves the links in the XML documentation into type names.
+    /// </summary>
+    /// <param name="node">The target node to process crefs in.</param>
+    /// <param name="elementName">The node type to process crefs for, can be `see` or `seealso`.</param>
+    private static void ResolveLangKeyword(XNode node, string elementName)
+    {
+        if (node == null || string.IsNullOrEmpty(elementName))
+        {
+            return;
+        }
+
+        var attributeName = DocumentationCommentXmlNames.LangwordAttributeName;
+
+        var nodes = node.XPathSelectElements($"//{elementName}[@{attributeName}]").ToArray();
+        foreach (var item in nodes)
+        {
+            var langword = item.Attribute(attributeName).Value;
+            if (string.IsNullOrEmpty(langword))
+            {
+                continue;
+            }
+
+            // TODO Is this the right thing to do? For example C# would get `true`, `false`, `null`
```

Comment:
@captainsafia any thoughts on this so I can get this PR over the line?

Reply:
WRT to normalization, I think the [C# keywords](https://learn.microsoft.com/en-us/dotnet/csharp/language-reference/keywords/) are probably sufficient to include in the document as-is.

WRT to VB in particular, I'm inclined to say we should consider limiting this generator to just C# (`[Generator(LanguageNames.CSharp)]`) since I haven't done any validation on VB-based projects and it would be a good safety-guard for anywhere we are doing CSharp specific syntax checks.

In practice, I think the keywords most likely to appear in API comments are probably things like true/false/null.

Reply:
Have opened #62069 with the proposed change to disable anything but C#.